webserver/server/app/models/delegation/delegationsForPool.sql

```diff
@@ -0,0 +1,20 @@
+/* 
+@name sqlStakeDelegationByPool
+@param pools -> (...)
+*/
+SELECT 
+	encode(credential, 'hex') as credential,
+	encode("Transaction".hash, 'hex') as tx_id,
+	COALESCE("StakeDelegationCredentialRelation".pool_credential IN :pools!, false) as is_delegation
+FROM "StakeDelegationCredentialRelation"
+JOIN "StakeCredential" ON stake_credential = "StakeCredential".id
+JOIN "Transaction" ON "Transaction".id = "StakeDelegationCredentialRelation".tx_id
+JOIN "Block" ON "Transaction".block_id = "Block".id
+WHERE 
+    (
+		"StakeDelegationCredentialRelation".pool_credential IN :pools! OR
+	 	"StakeDelegationCredentialRelation".previous_pool IN :pools!
+	) AND
+	"Block".slot > :min_slot! AND
+	"Block".slot <= :max_slot!
```

Comment:
we may want to have a way to paginate this in terms of number of entries in this range? Maybe by using the tx as a cursor, I need to check how hard would be it to re-use the existing pagination code for this.

Reply:
Typical pagination cursors do not work since they don't generally expect rollbacks. If you use a pagination cursor, it has to be set up so that if the underlying data changes in the middle of pagination you do not get in an invalid state